tests/unit/s2n_self_talk_ktls_test.c

```diff
@@ -270,6 +275,7 @@ int main(int argc, char **argv)
         };
 
         /* Test: s2n_sendfile */
+        file = open(argv[0], O_RDONLY);
```

Comment:
Can you avoid opening and closing this file twice?

Reply:
Yes I can. There is only one place in the test that uses `file`, which is to test s2n_sendfile with KTLS. Hence, I can move file opening and pread portion to the relevant location of the test. I also close the file after it is used to prevent file descriptor leaks.